pctasks/core/pctasks/core/cosmos/settings.py

```diff
@@ -46,35 +46,23 @@ def section_name(cls) -> str:
     max_bulk_put_size: int = 250
 
     def get_workflows_container_name(self) -> str:
-        if self.test_container_suffix:
-            return f"tmp-{self.workflows_container_name}-{self.test_container_suffix}"
-        return self.workflows_container_name
+        return f"{self.workflows_container_name}{self.test_container_suffix}"
 
     def get_workflow_runs_container_name(self) -> str:
-        if self.test_container_suffix:
-            return (
-                f"tmp-{self.workflow_runs_container_name}"
-                f"-{self.test_container_suffix}"
-            )
-        return self.workflow_runs_container_name
+        return f"{self.workflow_runs_container_name}{self.test_container_suffix}"
 
     def get_storage_events_container_name(self) -> str:
-        if self.test_container_suffix:
-            return (
-                f"tmp-{self.storage_events_container_name}"
-                f"-{self.test_container_suffix}"
-            )
-        return self.storage_events_container_name
+        return f"{self.storage_events_container_name}{self.test_container_suffix}"
 
     def get_items_container_name(self) -> str:
-        if self.test_container_suffix:
-            return f"tmp-{self.items_container_name}" f"-{self.test_container_suffix}"
-        return self.items_container_name
+        return f"{self.items_container_name}{self.test_container_suffix}"
 
     def get_records_container_name(self) -> str:
-        if self.test_container_suffix:
-            return f"tmp-{self.records_container_name}-{self.test_container_suffix}"
-        return self.records_container_name
+        return f"{self.records_container_name}{self.test_container_suffix}"
+
+    @validator("test_container_suffix", always=True)
+    def _validate_test_container_suffix(cls, v: Optional[str]) -> str:
+        return v or ""
```

Comment:
I think this is fine here, but I've been bitten before by validators changing the values of properties and that being super confusing during debugging/tracing logic. In that case it was a more substantial change (I think changed the type from the stated type), but in this case you're just ensuring its a string value for internal logic of the methods. Might be worth a comment to say that a validated model will never actually have a None value.